### ISSUE TITLE
i cant see miners

### ISSUE THREAD

**Issue by yigit311** [UnamSanctam/SilentCryptoMiner#2077]
![Screenshot_2](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/689aaa7f-e8aa-4259-bbc4-be277e4ac35d)
![Screenshot_3](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/73bc09bc-ade8-4f5b-b398-cefa1f54f0cd)
i cant see miners

**Comment by UnamSanctam** [comment]
You have SSL enabled (stratums) but you are using the non-SSL port 1010, for SSL you should use port 11010.

**Comment by yigit311** [comment]
etc.2miners.com:11010 rigth?

**Comment by UnamSanctam** [comment]
Yes that's correct.

**Comment by yigit311** [comment]
I did but things didn't change
![Screenshot_44](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/a5deb8d7-87f8-4730-a2b2-12c5577910e7)
![Screenshot_1](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/8a7a6088-35c4-447f-8b91-55340703bd52)

**Comment by UnamSanctam** [comment]
You need to find a share first before anything appears in the pool. You can check if your GPU usage has gone up using something like MSI Afterburner.

**Comment by yigit311** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/446aa74c-e2b5-430a-8a24-f198a82129c3)

**Comment by UnamSanctam** [comment]
Oh, you changed `stratums` to `stratum`, if you're going to use the SSL port 11010 then you should use `stratums`

**Comment by yigit311** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/eb497258-866e-4a75-9881-1eeeb3d082e3)
i think its working

**Comment by yigit311** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/105492169/1736942f-6973-4beb-bb76-bd72c4dc8cca)

**Comment by UnamSanctam** [comment]
Yes that looks like it's working, when you find a share it should appear in the pool (their statistics might have to update first too).

**Comment by yigit311** [comment]
THANKS İTS WORKİNG

**Comment by UnamSanctam** [comment]
Great, good luck with your mining.
